cmip6_cloud/test_new_workflow.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<p>@dgergel, when running this notebook on a local jupyter server, this cell throws a <code>FileNotFoundError</code> with the following Traceback:</p><p> </p><p>If I'm running locally, is this expected? If so, where I should I execute this notebook?</p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
&lt;ipython-input-7-6fcbd9f585cc&gt; in &lt;module&gt;
     20     dESGF = pd.concat(df_list,sort=False)
     21     dESGF = dESGF.drop_duplicates(subset =["file_name","version","checksum"])
---&gt; 22     dESGF.to_csv(os.path.join(zarr_local, 'ESGF_specific.csv'),index=False)
     23 else:
     24     dESGF = pd.read_csv('csv/ESGF_specific.csv')

~/.pyenv/versions/anaconda3-2019.10/envs/pangeo-forge3.8/lib/python3.8/site-packages/pandas/core/generic.py in to_csv(self, path_or_buf, sep, na_rep, float_format, columns, header, index, index_label, mode, encoding, compression, quoting, quotechar, line_terminator, chunksize, date_format, doublequote, escapechar, decimal, errors, storage_options)
   3385         )
   3386 
-&gt; 3387         return DataFrameRenderer(formatter).to_csv(
   3388             path_or_buf,
   3389             line_terminator=line_terminator,

~/.pyenv/versions/anaconda3-2019.10/envs/pangeo-forge3.8/lib/python3.8/site-packages/pandas/io/formats/format.py in to_csv(self, path_or_buf, encoding, sep, columns, index_label, mode, compression, quoting, quotechar, line_terminator, chunksize, date_format, doublequote, escapechar, errors, storage_options)
   1081             formatter=self.fmt,
   1082         )
-&gt; 1083         csv_formatter.save()
   1084 
   1085         if created_buffer:

~/.pyenv/versions/anaconda3-2019.10/envs/pangeo-forge3.8/lib/python3.8/site-packages/pandas/io/formats/csvs.py in save(self)
    226         """
    227         # apply compression and byte/text conversion
--&gt; 228         with get_handle(
    229             self.filepath_or_buffer,
    230             self.mode,

~/.pyenv/versions/anaconda3-2019.10/envs/pangeo-forge3.8/lib/python3.8/site-packages/pandas/io/common.py in get_handle(path_or_buf, mode, encoding, compression, memory_map, is_text, errors, storage_options)
    640                 errors = "replace"
    641             # Encoding
--&gt; 642             handle = open(
    643                 handle,
    644                 ioargs.mode,

FileNotFoundError: [Errno 2] No such file or directory: '/gcs/impactlab-data/climate/source_data/CMIP6/ESGF_specific.csv'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/cmip6-pipeline/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>